src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/ConsignmentFields.scala

```diff
@@ -266,4 +273,18 @@ object ConsignmentFields {
       tags = List(ValidateUserHasAccessToConsignment(UpdateConsignmentSeriesIdArg), ValidateUpdateConsignmentSeriesId)
     )
   )
+
+  private def getConsignmentsArgs(ctx: Context[ConsignmentApiContext, Unit]): (Int, Option[String], Option[ConsignmentFilter]) = {
+
+    val userId: Option[UUID] = ctx.args.argOpt("userId")
+    val limit: Int = ctx.args.arg("limit")
+    val currentCursor = ctx.args.argOpt("currentCursor")
+    val reportingAccess = ctx.ctx.accessToken.reportingRoles.contains(reportingRole)
+    val consignmentFilter = if (reportingAccess) {
```

Comment:
Even if a user has "reporting access", they still should also have the ability to filter by a user id if they need to. Doing this will prevent that.

Reply:
Yeah, that makes sense. I have removed the method itself and passed the `consignmentFilters` to the service.